src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -808,14 +808,29 @@ CLParser::ParseCommandLine(int argc, char** argv)
       case 0:
         params_->streaming = true;
         break;
-      case 1:
-        params_->max_threads = std::atoi(optarg);
-        params_->max_threads_specified = true;
+      case 1: {
+        std::string max_threads{optarg};
+        if (std::stoi(max_threads) > 0) {
+          params_->max_threads = std::stoi(max_threads);
+          params_->max_threads_specified = true;
+        } else {
+          Usage("maximum number of threads must be > 0");
+        }
         break;
-      case 2:
-        params_->sequence_length = std::atoi(optarg);
+      }
+      case 2: {
+        std::string sequence_length{optarg};
+        if (std::stoi(sequence_length) > 0) {
+          params_->sequence_length = std::stoi(sequence_length);
+        } else {
+          std::cerr
```

Comment:
The old code you removed elsewhere explicitly set the value to 20 in this case. Is 20 specified elsewhere as the default, and that code was redundant?